Examples/KitbotDemoBasic/src/main/java/frc/robot/Subsystems/DrivetrainSubsystem.java

```diff
@@ -32,7 +32,7 @@ private void setVoltages(double left, double right) {
   }
 
   public CommandBase setVoltagesCommand(DoubleSupplier left, DoubleSupplier right) {
-    return new RunCommand(() -> this.setVoltages(left.getAsDouble(), right.getAsDouble()), this);
+    return new RunCommand(this.setVoltages(left.getAsDouble(), right::getAsDouble), this);
```

Comment:
This seems wrong. RunCommand is being passed a value not a function. And one of the two values to setVoltages is a function while the other is a value. Either acquiring left should be passed as a function or the whole thing should still be a lambda, otherwise we're binding to a constant parameter.